components/claim-mgt/org.wso2.carbon.claim.mgt.ui/src/main/resources/web/claim-mgt/update-claim.jsp

```diff
@@ -84,14 +85,14 @@
                     }
                 }
                 function removeItem(dialect, claim, length) {
-                    var defaultDialect = "<%=UserCoreConstants.DEFAULT_CARBON_DIALECT%>";
+                    var defaultDialect = "<%=Encode.forJavaScriptBlock(UserCoreConstants.DEFAULT_CARBON_DIALECT)%>";

```

Comment:
Not necessary to encode here. This is a trusted constant without special characters